### DISCUSSION TITLE
What's the ` symbol in the examples on the front page?

### DISCUSSION THREAD

**Discussion by mavavilj** [hylang/hy#2433] [Q&A] [answered]
What's the ` symbol in the examples on the front page?

(defmacro do-while [test #* body]
  `(do
    ~@body
    (while ~test
      ~@body)))

(setv x 0)
(do-while x
  (print "Printed once."))

so **`**(do

https://hylang.org/

**Comment by mavavilj** [comment, marked as answer]
Answer is in docs, but could've been linked on the front page before using advanced constructs like that.

https://docs.hylang.org/en/stable/api.html#quasiquote

**Reply by Kodiologist** [reply]
I'm not sure what you're asking for. The manual as a whole is already linked, from the top of the page. Do you want the backtick in the example code to be a hyperlink? Do you want a sentence of prose about quasiquoting inserted in the table of examples, just before the quasiquote example?

**Reply by mavavilj** [reply]
As a newcomer, I thought it was a typo at first. I could intuitively understand the other parts of the code, but not that particular symbol.
